operator/src/main/resources/scripts/utils.py

```diff
@@ -61,3 +61,5 @@ def trace(arg1,arg2='SENTINEL'):
     traceInner('FINE',arg1)
   else:
     traceInner(arg1,arg2)
+
```

Comment:
It looks like there is no substantive change to this file. If so, can you revert?

Reply:
Done